examples/tree_2d_dgsem/elixir_euler_sedov_blast_wave_sc_subcell.jl

```diff
@@ -53,7 +57,7 @@ solver = DGSEM(basis, surface_flux, volume_integral)
 coordinates_min = (-2.0, -2.0)
 coordinates_max = (2.0, 2.0)
 mesh = TreeMesh(coordinates_min, coordinates_max,
-                initial_refinement_level = 3,
+                initial_refinement_level = 5,
```

Comment:
Why the increase in the number of cells? Was this intentional, and if yes, it is still "small enough" for CI testing?

Reply:
I thought for the elixir the resolution was too low to get a nice picture. The standard elixirs takes about 10 sek on my machine, which is okay in my opinion.

Reply:
But I did not change the CI test setup. It is still fixed to `initial_refinement_level=4`. This takes about 400ms on my machine.
In the CI runner it takes about 37 sec, where most of the time goes to the IO. For comparison, the standard sedov blast with element-wise limiting CI test runs for 22 sec within the CI run.

Reply:
What kind of I/O?

Reply:
Oh I think I overlooked and misunderstood something. The whole test set took about 36 seconds. But the timer output of the simulation itself just showed 2.24sec, where 53% of the time (now only 400ms) were the not tracked part of the I/O (~I/O~), see [here](https://github.com/trixi-framework/Trixi.jl/actions/runs/10901095979/job/30250163286#step:7:6054)

It seems like the subcell simulations take more time to set up. Comparing, the "existing" element-wise sedov blast test setup takes about 22sec, where 2.8sec is the simulation itself according to the timer.

Imo, that's somehow expected since I have relatively much cache variables which are set up before.

Reply:
ok